recipes/client.rb

```diff
@@ -31,6 +31,10 @@
   |a, b| a.name <=> b.name
 end
 
+if splunk_servers.empty?
+  splunk_servers = node['splunk']['splunk_servers']
+end
```

Comment:
Let's do an inline conditional.

And have a spec test that validates that this works.